src/oge/emissions.py

```diff
@@ -223,8 +223,18 @@ def identify_geothermal_generator_geotype(year):
     return geothermal_geotype
 
 
-def adjust_fuel_and_emissions_for_CHP(df):
-    """Allocates total emissions for electricity generation."""
+def adjust_fuel_and_emissions_for_CHP(df: pd.DataFrame) -> pd.DataFrame:
+    """Allocates total emissions for electricity generation.
+
+    Args:
+        df (pd.DataFrame): dataframe containing fuel and emissions columns mapped to
+        subplant_id. Must contain fuel_consumed_mmbtu,
+        fuel_consumed_mmbtu_for_electricity, and net_generation_mwh columns
```

Comment:
I believe in Google Style Python docstrings convention, if the description of a parameter spans multiple lines, then following lines should be indented.

Reply:
Thanks, I'll fix this throughout